quirks/change-password-URLs.json

```diff
@@ -65,6 +73,7 @@
     "minecraft.net": "https://www.minecraft.net/profile",
     "mojang.com": "https://www.minecraft.net/profile",
     "myaccount.ea.com": "https://myaccount.ea.com/cp-ui/security/index",
+    "naver.com": "https://nid.naver.com/user2/help/myInfo.nhn?m=viewChangePasswd&lang=ko_KR",
```

Comment:
Okay, one last bit. Should we remove the `lang` parameter? Does the link work without it?

Reply:
Yes it seems that the link works without it, so it will be nice if the `lang` parameter is removed.